app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -24,3 +24,4 @@
            "true",
            "false" %>
 <%= select_f f, :mode, compute_resource.disk_mode_types, :first, :last, { }, :class => "span5", :label => _("Disk mode"), :label_size => "col-md-2", :disabled => !new_host %>
+<%= select_f f, :controller_key, compute_resource.controller_ids, :first, :last, { }, :class => "span5", :label => _("SCSI Controller ID"), :label_size => "col-md-2" , :disabled => !new_host %>
```

Comment:
Yes, that would be wonderful. Everything we can get into core will be greatly appreciated. We (read: I) renamed `new_host` to `new_vm` as 637da2f2 allows you to create a host in foreman on an existing vm. So the naming was wrong.